### ISSUE TITLE
If namespace has underscore, route won't work

### ISSUE THREAD

**Issue by ileneruth** [ibm-cloud-architecture/microservices-netflix-eureka#12]
The .bluemixrc file specifies a hostname for the route that appends the namespace to 'netfix-eureka-', but if the namespace has an underscore, as mine did, then there's no error, but the route never shows up properly and you can't get to the eureka console.  Might be worth putting a warning somewhere, or having a more sophisticated script.  In my case, I hardcoded a different hostname.

**Comment by jesusmah** [comment]
you are right @ileneruth. The problem is with Bluemix network design which does not seem to like underscores in route hostnames at all... Thanks for bringing this up!
